docs/index.md

```diff
@@ -49,22 +53,28 @@ and it perfectly covers our needs.
 To install the plugin from the UI you just have to (other installation methods described
 [here][grafana-json-datasource-install]):
 
-1. Navigate to the *Configuration > Plugins* section.
+1. Navigate to the ***Configuration > Plugins*** section.
 ![Configuration > Plugins](./images/plugins.png)
 2. Search for the term “JSON API” so you should get the plugin developed by Marcus Olsson.
 ![JSON API Plugin](./images/json-api-plugin.png)
-3. Click on the *Install* button.
+3. Click on the ***Install*** button.
 ![JSON API install](./images/json-api-install.png)
-4. Click on the *Create a JSON API* data source button.
+4. Click on the ***Create a JSON API*** data source button.
 ![Create datasource](./images/create-datasource.png)
 5. Configure the data source with the following values:
    1. Name: System73 Analytics
    2. URL: [https://api.system73.com/analytics](https://api.system73.com/analytics)
-   3. Add a header on *Custom HTTP Headers* with the following:
-      1. Header: `Authorization`
-      2. Value: `Bearer <edge_analytics_api_key>` (Available on [System73 Portal][s73-portal])
+   3. Add a header on ***Custom HTTP Headers*** with the following:
+      1. Header:
+
+              Authorization
```

Comment:
code block

Reply:
This is a codeblock, mardownlint complains because its between a list so its 8 spaced code block

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks

> Tip: To preserve your formatting within a list, make sure to indent non-fenced code blocks by eight spaces.